app/services/api/google_analytics.rb

```diff
@@ -4,7 +4,7 @@
 module Api
   class GoogleAnalytics
     def register_event(event_action)
-      if A9n.host_ == 'aircasting.org'
+      if A9n.host_ == 'aircasting.habitatmap.org'
```

Comment:
I'm going to change this to something like `A9n.google_analytics == "enabled"`. Cause, I guess it's there so that this is not run on clone.